openedx/features/course_bookmarks/static/course_bookmarks/fixtures/bookmarks.html

```diff
@@ -0,0 +1,23 @@
+<div class="course-view container" id="course-container">
+    <header class="page-header has-secondary">
+        <div class="page-header-main">
+            <nav aria-label="Discussions" class="sr-is-focusable" tabindex="-1">
```

Comment:
Good catch! This is only in the fixture, but it shows that the name was wrong in the main template too.